grammar.js

```diff
@@ -244,7 +253,7 @@ module.exports = grammar({
       ';'
     )),
 
-    modifier: $ => prec.right(choice(
+    _modifier_no_ref: $ => prec.right(choice(
```

Comment:
Maybe we just call this modifier and remove the other two to avoid the conflict rules?